src/components/MainRouter/MainRouter.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { Redirect, Route, Switch } from "react-router";
+import Home from "../../pages/Home";
+import Login from "../../pages/Login/Login";
+import useAuth from "../../hooks/useAuth";
+import PrivateRoute from "../../components/PrivateRoute";
```

Comment:
Can we use the absolute path of files instead of the relative path?

Absolute paths are easy to maintain, readable. If you change the location (route) of the current file, you need to update all the paths imported in that file, which is again extra work.

Reply:
Great suggestion. I'll make a separate PR for it https://jira.crossroads.org.hk/browse/GCW-3434